sprout/appliances/models.py

```diff
@@ -717,6 +720,11 @@ class Power(object):
     name = models.CharField(max_length=64, help_text="Appliance's name as it is in the provider.")
     ip_address = models.CharField(max_length=45, null=True, help_text="Appliance's IP address")
 
+    openshift_ext_ip = models.CharField(max_length=45, null=True,
+                                        help_text="Openshift's project external ip")
+    openshift_project = models.CharField(max_length=45, null=True,
+                                         help_text="Openshift's project name")
+
```

Comment:
You need to generate and include the migration for this change.

Reply:
Of course, I'll prepare migration. I haven't included it because I doubt that my changes are enough

Reply:
done